sync/src/relayer/mod.rs

```diff
@@ -142,7 +142,25 @@ impl Relayer {
                 ban_time,
                 status
             );
+            metric!({
+                "measurement": "error",
+                "tags": {"source": item_name, "status": format!("{:?}", status.code()) },
+                "fields": {"count": 1},
```

Comment:
I noticed sometimes the fields are `count: 1` other time the fields just be empty, what's the difference?

Reply:
It's my fault that uses different style at different places.
I will add `{"count": 1}` to these empty fields. Thanks.

Reply:
Why not just empty the fields since the key pair have no actually meaning?

Reply:
Keep consistency. So that when integrates these metrics, we can sum the total events by simply `sum += metric["field"]["count"]`.

The field "count" will be used to sum the number of happened events. For example, `{ "topic": "fresh_transaction", "field": { "count" : 2 } }` means that we require `2` fresh transactions.

Reply:
IMO we should only keep the consistency under the same "topic". It's unavoidable to extend specific fields, for example:

```
                metric!({
                     "measurement": "reorg",
                     "tags": { "source": "chain", },
                     "fields": { "attached": fork.attached_blocks.len(), "detached": fork.detached_blocks.len(), },
                 });
```

The "reorg" also has no `count: 1` field.

For every "fresh_transaction" it makes sense to keep the key "count", but for "error" or "warn", the field is meaningless, just junk outputs to increase the log size.

Reply:
Got it!